README.md

```diff
@@ -14,12 +19,55 @@ Deze aanpak biedt aanzienlijke voordelen voor zowel gemeenten als burgers. Voor
+Een Schema bevat de beschrijving van hoe een Object er uit moet/kan zien. 
+Vul een naam, beschrijving en een unieke reference (url zoals: https://jouwDomein.nl/naamVanJouwSchema.schema.json) in en maak een Schema aan.
+
+Dan willen aan dit nieuwe Schema properties toevoegen via het tab -> `Properties` -> `Add Property`. 
+De properties zijn de verschillende velden wat een Object kan (of moet) hebben. 
+Deze velden kunnen later als kolommen getoond worden in een objecten tabel.
+
+> **Opmerking:**
+> Als je later via de Gateway UI een excel bestand van objecten wilt kunnen importeren is het verstandig om hier alleen velden van type = String toe te voegen aan je Schema.
+
+Om te regelen dat de objecten tabellen plugin via een API URL objecten uit de gateway kan ophalen moeten we een Endpoint toevoegen aan de Gateway via -> `Endpoints` -> `Add Endpoint` (Klik [hier](https://commongateway.github.io/CoreBundle/pages/Features/Endpoints) voor meer informatie over Endpoints).\
+Maak een Endpoint aan voor elke Schema die je hebt toegevoegd. Geef elk Endpoint een naam, unieke reference (url zoals: https://jouwDomein.nl/naamVanJouwEndpoint.endpoint.json), selecteer de Schema waarvoor je het Endpoint maakt en vink bij Methods "GET" aan.\
+Path en Path Regex bepalen hoe je Endpoint er uiteindelijk uit ziet, hier volgt een voorbeeld voor deze velden:
+
+Uiteindelijke GET url = https://jouwGatewayDomein/api/barendrecht/betalingen \
+Path = `barendrecht/betalingen/{id}` \
+Path Regex = `^barendrecht/betalingen/?([a-z0-9-]+)?$`
+
+### Objecten importen via de Common Gateway ###
+
+De Common Gateway ondersteunt het toevoegen van objecten door middel van bijvoorbeeld een Excel bestand. 
+Meer informatie over deze functionaliteit kun je [hier](https://commongateway.github.io/CoreBundle/pages/Features/ImportExport) terug vinden.
+
+Via de Gateway UI kun je via -> `Import and upload` bestanden uploaden. Hier onder volgen een aantal punten om op te letten bij het inladen van objecten via een excel bestand:
+- Voordat je een excel bestand import, zorg er voor dat in je excel sheet een kolom aanwezig is met een unieke identificatie voor elke regel / object (kan zo simpel zijn als nummering 1,2,3 etc.). 
+Voeg de naam van deze kolom toe als Property aan je Schema en aan de mapping die je gebruikt voor importeren:`'_id' : 'naamVanDeKolom'`. 
+Hierdoor kan je meerdere keren het zelfde bestand uploaden zonder dat er dubbelen objecten ontstaan.
+- Voordat je een excel bestand import, zorg dat je deze hebt opgeslagen met het juiste tabblad open.
+Alleen de gegevens uit dit tabblad worden gebruikt voor het importeren van objecten.
+- Voor optimale performance/ervaring raden we voor nu aan niet meer dan 300 tot 500 objecten per tabblad tegelijk in te laden (meer is zeker wel mogelijk en kan je ook zeker proberen, maar het kan zijn dat de Gateway UI dit niet zo leuk vind).
```

Comment:
(meer is mogelijk, maar de kans dat hierbij de browser vastloopt wordt groter naarmate het aantal objecten dat ingeladen moet worden stijgt).